alerts/alerts.md

```diff
@@ -81,6 +85,10 @@ Having verified that events are being detected and logged, we can now move on to
 Dispatcher searches should use either `table`, `text`, or `raw` as their renderer. If table is used, the alert will use the columns of the table as the "fields" of each event. If text or raw are used, the alert will use the enumerated values attached to each entry as fields.
 ```
 
+```{warning}
+You can select scheduled searches belonging to other users as dispatchers, but be aware that if the owner of the search decides to change it, your alert may stop working!
```

Comment:
I would consider re-wording this a bit. An admin can pick *any* scheduled search, but a non-admin user can't choose just *any* scheduled search belonging to another user - it has to be global or shared. In other words, they have to have been granted access to it. Maybe we should say something like:
"If you have been granted access to a scheduled search that you don't own, you will be able to select it as an alert dispatcher, but be aware that if the owner of the search decides to change it, your alert may stop working!"